src/calculateRentalCost.js

```diff
@@ -4,7 +4,21 @@
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
+  const dayPrice = 40;
+  const weekDiscount = 50;
+  const threeDayDiscount = 20;
+
+  let price = days * dayPrice;
+
+  if (days >= 7) {
+    price = price - weekDiscount;
+  }
```

Comment:
You can directly return the calculated price within the `if` block to simplify the function and avoid the need for the `price` variable. This adheres to the checklist item 4 and 5, which recommend using `if` with `return` over `if else`, and not adding `else` after `if` with `return`.